reference/pipeline/stages/index.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: single
+title:  "Pipeline Stages"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+> Note: this is a first pass at this documentation; more details will be added
+> soon.
+
+This article describes the currently-supported stages that you can add to your
+Spinnaker pipelines.
+
+## General
+
+### Bake
+Bakes an image in the specified region.
+
+### Canary Analysis
+Runs automated canary analysis against the deployment before fully deploying.
+For more information on automated canary analysis in Spinnaker, see the
+[user guide](/guides/user/canary/).
```

Comment:
I think you could delete this whole 2nd sentence and just link "automated canary analysis" in the first sentence, to this target.